exports/taskfiles/utils/checksum.yaml

```diff
@@ -25,58 +36,97 @@ tasks:
       # input patterns cannot be quoted since they're evaluated by the shell and the results are
       # passed to `tar` as arguments. If the input patterns are passed to `tar` with quotes, the
       # pattern won't be evaluated and will instead be treated literally.
-      - >-
-        tar
-        --create
-        --file -
-        --group 0
-        --mtime "UTC 1970-01-01"
-        --numeric-owner
-        --owner 0
-        --sort name
-        --no-anchored
-        --wildcards
-        {{- range .EXCLUDE_PATTERNS}}
-        --exclude="{{.}}"
-        {{- end}}
-        {{- range .INCLUDE_PATTERNS}}
-        {{.}}
-        {{- end}}
-        2> /dev/null
-        | md5sum > {{.CHECKSUM_FILE}}
-    # Ignore errors so that dependent tasks don't fail
-    ignore_error: true
+      - defer: "rm -f '{{.TMP_ERR_LOG}}'"
+      - |-
+        if ! \
+            {{.ARCHIVER}} \
+            --create \
+            --file - \
+            --group 0 \
+            --mtime "UTC 1970-01-01" \
+            --numeric-owner \
+            --owner 0 \
+            --sort name \
+            --no-anchored \
+            --wildcards \
+            {{- range .EXCLUDE_PATTERNS}}
+            --exclude="{{.}}" \
+            {{- end}}
+            {{- range .INCLUDE_PATTERNS}}
+            {{.}} \
+            {{- end}}
+            2> "{{.TMP_ERR_LOG}}" \
+                | md5sum > "{{.CHECKSUM_FILE}}" \
```

Comment:
indent
```suggestion
          {{.ARCHIVER}} \
          --create \
          --file - \
          --group 0 \
          --mtime "UTC 1970-01-01" \
          --numeric-owner \
          --owner 0 \
          --sort name \
          --no-anchored \
          --wildcards \
          {{- range .EXCLUDE_PATTERNS}}
          --exclude="{{.}}" \
          {{- end}}
          {{- range .INCLUDE_PATTERNS}}
          {{.}} \
          {{- end}}
          2> "{{.TMP_ERR_LOG}}" \
            | md5sum > "{{.CHECKSUM_FILE}}" \
```

Reply:
or maybe the 4-space indents are intentional? i can't be sure what are the conventions when we mix shell commands with go templates...

the code is fairly readable either way anyways